cli/command/container/kill.go

```diff
@@ -32,7 +32,7 @@ func NewKillCommand(dockerCli command.Cli) *cobra.Command {
 	}
 
 	flags := cmd.Flags()
-	flags.StringVarP(&opts.signal, "signal", "s", "KILL", "Signal to send to the container")
+	flags.StringVarP(&opts.signal, "signal", "s", "SIGKILL", "Signal to send to the container")
```

Comment:
I'm considering to remove this default from the client altogether, because the daemon already has a default. WDYT?

Reply:
done 👍